examples/main.go

```diff
@@ -5,32 +5,52 @@ import (
 	"fmt"
 	"log"
 
+	"sync"
+
 	"github.com/nuveo/gofn"
+	"github.com/nuveo/gofn/iaas/digitalocean"
 	"github.com/nuveo/gofn/provision"
 )
 
 func main() {
-
+	wait := &sync.WaitGroup{}
 	contextDir := flag.String("contextDir", "./", "a string")
 	dockerfile := flag.String("dockerfile", "Dockerfile", "a string")
 	imageName := flag.String("imageName", "", "a string")
 	remoteBuildURI := flag.String("remoteBuildURI", "", "a string")
 	volumeSource := flag.String("volumeSource", "", "a string")
 	volumeDestination := flag.String("volumeDestination", "", "a string")
 	flag.Parse()
+	wait.Add(1)
+	run(*contextDir, *dockerfile, *imageName, *remoteBuildURI, *volumeSource, *volumeDestination, wait)
+	wait.Add(1)
+	run(*contextDir, *dockerfile, *imageName, *remoteBuildURI, *volumeSource, *volumeDestination, wait)
+	wait.Add(1)
+	run(*contextDir, *dockerfile, *imageName, *remoteBuildURI, *volumeSource, *volumeDestination, wait)
+	wait.Add(1)
+	run(*contextDir, *dockerfile, *imageName, *remoteBuildURI, *volumeSource, *volumeDestination, wait)
+	wait.Add(1)
+	run(*contextDir, *dockerfile, *imageName, *remoteBuildURI, *volumeSource, *volumeDestination, wait)
```

Comment:
Remember the principal DRY... Don't repeat yourself. Do you really need to copy and paste 5 times the same code? https://en.wikipedia.org/wiki/Don't_repeat_yourself